ci/codespell.ignores

```diff
@@ -15,6 +15,7 @@ reord
 mimick
 cloneable
 keypair
+keypairs
```

Comment:
Shouldn't be needed.  If you're seeing this locally then perhaps you need to upgrade your codespell version?

Reply:
I should have the latest one as I have install it through pip3 install.
My version is the 2.2.1.